.devcontainer/images/.claude/commands/init/discovery.md

```
@@ -0,0 +1,233 @@
+  decision:
+    if_template_detected:
+      action: "Run Phase 1 (Discovery Conversation)"
+      message: "Template detected. Let's discover your project."
+    if_personalized:
+      action: "Skip to Phase 4 (Validation)"
+      message: "Project already personalized. Validating..."
+```
+
+**Output Phase 0 (other project - reset):**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)